CMakeLists.txt

```diff
@@ -345,6 +345,7 @@ if(WITH_LZ4)
 endif(WITH_LZ4)
 
 option(WITH_CEPH_DEBUG_MUTEX "Use debug ceph::mutex with lockdep" OFF)
+option(WITH_DEBUG_PG_REFS "Debug PG references" OFF)
```

Comment:
we already have an option for this

https://github.com/ceph/ceph/blob/c1112928b7926fdc0a53d254a5f72055728f3b58/CMakeLists.txt#L572

which sets 

https://github.com/ceph/ceph/blob/c1112928b7926fdc0a53d254a5f72055728f3b58/src/include/config-h.in.cmake#L280-L281